lib/segment/src/index/sparse_index/sparse_vector_index.rs

```diff
@@ -364,15 +368,14 @@ impl<TInvertedIndex: InvertedIndex> VectorIndex for SparseVectorIndex<TInvertedI
     }
 
     fn files(&self) -> Vec<PathBuf> {
-        let config_file = SparseIndexConfig::get_config_path(&self.path);
-        if !config_file.exists() {
-            return vec![];
-        }
-
-        let mut all_files = vec![];
-        all_files.push(config_file);
-        all_files.extend_from_slice(&self.inverted_index.files());
-        all_files
+        [
+            SparseIndexConfig::get_config_path(&self.path),
+            InvertedIndexMmap::index_file_path(&self.path),
```

Comment:
Why call directly the concrete `InvertedIndexMmap` type instead of going through the trait?
This is seems to be breaking encapsulation.

Reply:
Is it fine for you?

Reply:
Let's give it a try :+1:

Reply:
Tried, https://github.com/qdrant/qdrant/pull/3154/commits/9f459e67f2b1e07ead364ce3363d2136e2bace6e